.github/workflows/update.yml

```
@@ -14,17 +14,17 @@ jobs:
           node-version: 18
       - run: npm ci
       - run: npm run download
-      - uses: actions/checkout@v4
-        with:
-          repository: xuorig/anicca
-          path: anicca
-      - uses: actions-rs/toolchain@88dc2356392166efad76775c878094f4e83ff746
-        with:
-          toolchain: stable
+      # - uses: actions/checkout@v4
+      #   with:
+      #     repository: xuorig/anicca
+      #     path: anicca
+      # - uses: actions-rs/toolchain@88dc2356392166efad76775c878094f4e83ff746
+      # with:
+      #   toolchain: stable
```

I would recommend we add a comment on why this is commented out
```suggestion
      # Temporary disabled generating diff files
      # https://github.com/octokit/openapi/pull/394
      # - uses: actions/checkout@v4
      #   with:
      #     repository: xuorig/anicca
      #     path: anicca
      # - uses: actions-rs/toolchain@88dc2356392166efad76775c878094f4e83ff746
      # with:
      #   toolchain: stable
```